src/ipc-events.ts

```diff
@@ -0,0 +1,7 @@
+export enum IpcEvents {
+  CLICK_TITLEBAR_MAC = 'CLICK_TITLEBAR_MAC'
+}
+
+export const ipcMainEvents = [
```

Comment:
I don't think we're using this variable here. I think it's fine if we don't define a separate an array for `ipcMainEvents`.

Reply:
Took it out, thanks!